sdk/core/azure-core/test/ut/http_test.cpp

```diff
@@ -121,6 +121,24 @@ namespace Azure { namespace Core { namespace Test {
         (std::pair<std::string, std::string>("valid3", "header3")));
   }
 
+  TEST(TestHttp, RawResponseHttpVersions)
+  {
+    Http::RawResponse responseHttp10(1, 0, Http::HttpStatusCode::Accepted, "Test");
```

Comment:
We should parameterize this test so that we can test all scenarios, positive cases and negative cases.

Reply:
We have no negative case validation in place.

What do you expect as the behavior in those cases?

Reply:
I would expect invalid values to throw in the Raw response ctor. Until then, I don't want to codify in tests the current behavior for such inputs.